src/utils/viewer/viewer.cpp

```diff
@@ -524,6 +525,10 @@ Viewer::Viewer(const Arguments& arguments)
     Mn::Debug{} << "Stage using DEFAULT_LIGHTING_KEY";
     simConfig.sceneLightSetup = esp::DEFAULT_LIGHTING_KEY;
   }
+  if (!args.value("mesh-simplification").empty()) {
+    simConfig.meshSimplificationFraction =
+        (float)std::stoi(args.value("mesh-simplification")) / 100;
```

Comment:
FYI, you can do this:

```suggestion
        args.value<Float>("mesh-simplification") / 100;
```